### PR TITLE
Update model-attributes.rst

### PR DESCRIPTION
**Pull request by tranfuga25s**
Warning using models with recursive = -1 and events in related  models.

### CODE REVIEW

File: en/models/model-attributes.rst [hunk 1]
```diff
@@ -140,6 +140,11 @@ the default recursive level is 1.
 
         public $recursive = -1;
 
+.. tip::
+
+   If you use events in your system, using the value -1 for recursive will
+   disable all event triggering in the associated model. This appens because
```
Comment (dereuromark) [inline en/models/model-attributes.rst]: typo happens

File: en/models/model-attributes.rst [hunk 2]
```diff
@@ -140,6 +140,11 @@ the default recursive level is 1.
 
         public $recursive = -1;
 
+.. tip::
+
+   If you use events in your system, using the value -1 for recursive will
+   disable all event triggering in the associated model. This happens because
+   no relation is created when the value is set to -1.
```
Comment (markstory) [inline en/models/model-attributes.rst]: is should probably be 'are' here.

Comment (tranfuga25s) [inline en/models/model-attributes.rst]: Sorry, I'm not a english native speaker...
Do you mean "no relations are created when the value is set to -1"??

Comment (dereuromark) [inline en/models/model-attributes.rst]: Yes :)

**Pull request merged**